aspect/intellij_info_impl.bzl

```diff
@@ -354,7 +354,14 @@ def _collect_generated_go_sources(target):
         files = target[OutputGroupInfo].go_generated_srcs.to_list()
         return [f for f in files if f.basename.endswith(".go")]
     else:
-        return None
+        files = target.files.to_list()
+        return [f for f in files if f.basename.endswith(".go")]
+
+def _collect_go_sources(target):
+    """Returns a depset of go source files generated by this target."""
+    if hasattr(target, "files"):
+        files = target.files.to_list()
+        return [f for f in files if f.basename.endswith(".go")]
```

Comment:
Indeed, This line will collect the go file only to avoid those .a or .s file to broke indexing.